test/refactoring/RefactoringKind/basic.swift

```diff
@@ -174,6 +174,13 @@ func testStringLiteral() -> String {
   return "abc"
 }
 
+func testCollapseNestedIf() {
+  let a = 3
+  if a > 2 {
+    if a < 10 {}
+  }
+}
+
```

Comment:
can we add a test case to ensure such refactoring kind doesn't show up in a multi-statement brace statement? 
e.g 
```
if a > 2 { // not here.
  if a < 5 {
  }
  if a < 10 {
  }
}
```

Reply:
oh, I saw your listed to do above. Please ignore this.